prov/verbs/src/fi_verbs.h

```diff
@@ -429,6 +433,9 @@ struct vrb_mem_desc {
 	enum fi_hmem_iface	iface;
 };
 
+#define MR_DESC_LKEY(desc) (((struct vrb_mem_desc *)(desc))->mr->lkey)
+#define MR_DESC_IFACE(desc) (((struct vrb_mem_desc *)(desc))->iface)
```

Comment:
Use VERBS prefix.  Though I would consider just dropping both of these macros.  The LKEY macro is only used in one location.  IFACE is only used in 2.

Reply:
I'll drop marcos.